src/components/Search/CompareButton.tsx

```diff
@@ -1,28 +1,19 @@
 import Button from '@mui/material/Button';
-import { style } from 'typestyle';
-
-import { useAppSelector } from '../../hooks/app';
-import { ButtonStyles } from '../../styles';
 
 interface CompareButtonProps {
   label: string;
 }
 
 export default function CompareButton({ label }: CompareButtonProps) {
-  const mode = useAppSelector((state) => state.theme.mode);
-  const btnStyles = ButtonStyles(mode);
-
-  const styles = {
-    button: style({ ...btnStyles.Primary }),
-  };
-
   return (
     <Button
       id='compare-button'
       variant='contained'
-      className={`compare-button ${styles.button}`}
-      sx={{ textTransform: 'none !important' }}
+      color='primary'
       type='submit'
+      sx={{
+        height: 32,
+      }}
```

Comment:
This was in the shared button styles that I removed from the button as well.
I think the sizing is usually a "local" css, in that it can be different in all different cases. But if you disagree please tell me.
For example maybe the protocol theme says that it should always be 32 px high. If necessary I think we could implement that globally using global classes on MuiButton too.

Reply:
Yeah, these global buttons styles like height should be in protocol theme so that we maintain the` buttons `standard set by Dasha in the[ figma](https://www.figma.com/design/LRV76asSVOULlsWbRNZYp0/PerfCompare?node-id=2835-113706&m=dev) file.